### PR TITLE
Add post release documentation

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: doc/developer/build_release.rst [hunk 1]
```diff
@@ -188,3 +188,21 @@ Push your changes:
 
    When you push a tag with the pattern `^[0-9].[0-9]+.[0-9]$` a new release
    will automatically be created on Travis CI.
+
+Post release tasks
+------------------
+
+When a new release or a new version is don you should do the following tasks:
```
Comment (ochriste) [inline doc/developer/build_release.rst]: done

Comment (sbrunner) [inline doc/developer/build_release.rst]: done

File: doc/developer/build_release.rst [hunk 2]
```diff
@@ -188,3 +188,21 @@ Push your changes:
...
    will automatically be created on Travis CI.
+
+Post release tasks
+------------------
+
+When a new release or a new version is done you should do the following tasks:
+
+ * Merge the release changes (on cgxp and on c2cgeoportal)
+   to the upper branches i.e.: 1.6 => 2.0, 2.0 => master.
+ * Upgrade the demo in your home folder with the c2cool.
```
Comment (ger-benjamin) [inline doc/developer/build_release.rst]: ...with the *c2ctool* ?

Comment (sbrunner) [inline doc/developer/build_release.rst]: yes :-)

Comment (sbrunner) [inline doc/developer/build_release.rst]: done

**Comment by ger-benjamin** [comment]
Not tested but syntactically correct.

**Pull request merged**